aws-serverless-java-container-core/src/main/java/com/amazonaws/serverless/proxy/internal/servlet/AwsHttpServletRequest.java

```diff
@@ -370,7 +370,11 @@ protected String generateQueryString(MultiValuedTreeMap<String, String> paramete
                     // TODO: Should we concatenate the rest of the values?
                     if (newValue.getValue() == null) {
                         newValue.setKey(kv[0].trim());
-                        newValue.setValue(kv[1].trim());
+                        if (kv.length == 1) {
+                            newValue.setValue("");
```

Comment:
Should this be empty or null? Since the value just wasn't there I would probably opt for null